paimon-core/src/main/java/org/apache/paimon/table/source/DataSplit.java

```diff
@@ -75,6 +77,11 @@ public boolean isStreaming() {
         return isStreaming;
     }
 
+    public Optional<DataFileMeta> getLatestFile() {
+        return this.dataFiles.stream()
+                .max(Comparator.comparingLong(f -> f.creationTime().getMillisecond()));
```

Comment:
Why not return max `creationTimeEpochMillis` directly?

Reply:
I think it'll bring more calculation that is meanless. Each comparation will calculate the localtime to epoch.